prime-router/metadata/organizations.yml

```diff
@@ -19,6 +19,11 @@
         jurisdictionalFilter: { standard.testing_lab_state: AZ }
 # OLD        jurisdictionalFilter: { standard.patient_state: AZ }
         transforms: { deidentify: false }
+        batch:
+          operation: MERGE
+          numberPerDay: 1440 # Every minute
+          initialBatch: 00:00
+          timeZone: ARIZONA
         address: http://localhost:1181/
```

Comment:
For now, while we are debugging, send every minute. Will change once we onboard and a regular cadence is established.

Reply:
If possible - might want to switch to a cron like schema (e.g what if we want to send every 15 minutes or every hours between 0 and 8am?)